frappe/patches/v7_2/patch_notes_notify_on_every_login.py

```diff
@@ -0,0 +1,8 @@
+import frappe
+
```

Comment:
please remove this patch too... (it was not in patches.txt) so would not have run anyways...